src/Security/Authorization/Voter/UserAclVoter.php

```diff
@@ -39,7 +39,7 @@ public function supportsAttribute($attribute)
      */
     public function vote(TokenInterface $token, $object, array $attributes)
     {
-        if (!$this->supportsClass(get_class($object))) {
+        if (!is_object($object) || !$this->supportsClass(get_class($object))) {
```

Comment:
Please change `$object` to `$subject` everywhere in that method, otherwise it hard to understand.